docs/notebooks/entropic_map_estimator.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #30.    plt.scatter(y2[0:500,0],y2[0:500,1],c='b',marker='+',label='Target')
</pre><p>Would you call this exactly the target? it sounds a bit different than the i.i.d target sample in the first plot (but maybe I am too picky here)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/175/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>